### PR TITLE
blog: add part3 of drone blog

### PR DESCRIPTION
**Pull request by ric96**
Signed-off-by: Sahaj Sarup <sahaj.sarup@linaro.org>

### CODE REVIEW

File: _posts/2020-03-24-96boards-drone-project.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+image:
+    featured: true
+    path: /assets/images/blog/drone.png
+    name: drone.png
+categories: blog
+series: "96boards Drone Project"
+tags: 64-bit, 96Boards, aarch64, ARM, ARMv8, Consumer Edition, Hikey970, Linaro, Linux, arm64, real time, drones, aerocore2
+---
+
+In this blog I'll like to cover the hardware and the software specifics of the drone project.
```
Comment (sgermanserrano) [inline _posts/2020-03-24-96boards-drone-project.md]: typo
```suggestion
In this blog I'd like to cover the hardware and the software specifics of the drone project.
```

File: _posts/2020-03-24-96boards-drone-project.md [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+        - Older PX4 commits
+        - PCB tracing
+        - Trial and error
+    - However Gumstix provided as much help as possible.
+
+## Firmware Design
+
+- **ESC Initialization**
+    - The ESC has ONLY ONE INPUT, and that is the PWM signal that comes out of the receiver.
+    - The ESC does one of three things with the PWM
```
Comment (sgermanserrano) [inline _posts/2020-03-24-96boards-drone-project.md]: Mentions 3 things but there are 4 sub-bullets

**Comment by ric96** [comment]
@kylekirkby how can we resize the images?

**Comment by kylekirkby** [comment]
> @kylekirkby how can we resize the images?

I'll look into this for you asap. I know that, currently, the img-repsonsive class is being applies to images so that they take up the full width of their container. I'll need to add some additional functionality

**Pull request merged**